src/core/plugins/spec/selectors.js

```diff
@@ -494,12 +494,38 @@ export const validationErrors = (state, pathMethod) => {
   let paramValues = state.getIn(["meta", "paths", ...pathMethod, "parameters"], fromJS([]))
   const result = []
 
```

Comment:
How about doing the following?

```js
if (paramValues.length === 0) return result
```

Reply:
Is this to make it more clear what's happening if we don't have these values?
If so, makes sense to me.

Reply:
> Is this to make it more clear what's happening if we don't have these values?

Well primary factor is performance here. This is not a memoized selector, which means it's executing always fully. We want to return as soon as we can to avoid logic of creating inner functions and performing the reductions as soon as possible.